### PR TITLE
Fix inconsistent yaml indentation

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -7,34 +7,34 @@ env:
 
 jobs:
   ci-config:
-      runs-on: ubuntu-latest
```
Comment (dscho) [inline .github/workflows/main.yml:10]: Looks good, but you will have to fix the closing bracket in your Signed-off-by: line, and you should prefix the commit subject with "ci" like so:

> ci: fix inconsistent YAML indentation

**Comment by atabaytebrizl** [comment]
14 Ağu 2020 Cum 03:05 tarihinde gitgitgadget-git[bot] <
notifications@github.com> şunu yazdı:

**Comment by atabaytebrizl** [comment]
15 Ağu 2020 Cmt 01:25 tarihinde gitgitgadget-git[bot] <
notifications@github.com> şunu yazdı:

**Pull request closed without merging**